docs/samplesheets/examples.md

```diff
@@ -44,22 +44,17 @@ tuple val(meta), path(fastq_1), path(fastq_2), path(bed)
 
 It may be necessary to manipulate this channel to fit your process inputs. For more documentation, check out the [Nextflow operator docs](https://www.nextflow.io/docs/latest/operator.html), however here are some common use cases with `.fromSamplesheet()`.
 
-## Using a samplesheet with no headers
+## Using a sample sheet with no headers
 
-Sometimes you only have one possible input in the pipeline samplesheet. In this case it doesn't make sense to have a header in the samplesheet. This can be done by creating a samplesheet with an empty string as input key:
+Sometimes you only have one possible input in the pipeline sample sheet. In this case it doesn't make sense to have a header in the sample sheet. This can be done by removing the `properties` section fro m the samplesheet and changing the type of the element from `object` to whatever type you want:
```

Comment:
```suggestion
Sometimes you only have one possible input in the pipeline sample sheet. In this case it doesn't make sense to have a header in the sample sheet. This can be done by removing the `properties` section from the sample sheet and changing the type of the element from `object` the desired type:
```